pkg/grpc/tracing/tracing.go

```diff
@@ -278,6 +278,7 @@ func GetProcessKprobe(event *MsgGenericKprobeUnix) *tetragon.ProcessKprobe {
 		kernelSymbols, err := ksyms.KernelSymbols()
 		if err != nil {
 			logger.GetLogger().WithError(err).Warn("stacktrace: failed to read kernel symbols")
+			continue
```

Comment:
I think we are using this in other versions... maybe worth a backport? @mtardy @kkourt

Reply:
I don't think is worth backporting, but happy to defer to @mtardy

Reply:
just saw that! indeed, out of curiosity, how you detected that?